overlord/aspectstate/aspectstate.go

```diff
@@ -49,9 +49,8 @@ func SetAspect(databag aspects.DataBag, account, bundleName, aspect, field strin
 	return nil
 }
 
-// GetAspect finds the aspect identified by the account, bundleName and aspect in the
-// supplied databag and returns the specified field's value through the "value"
-// output parameter.
+// GetAspect finds the aspect identified by the account, bundleName and aspect
+// and returns the specified field value through the value output parameter
```

Comment:
... field value from the provided matching databag through the value output parameter.